Orange/widgets/data/owpreprocess.py

```diff
@@ -506,7 +508,9 @@ def __init__(self, parent=None):
         self.__uni_fs.setItems(
             [{"text": "Information gain", "tooltip": ""},
              {"text": "Gain ratio"},
-             {"text": "Gini index"}
+             {"text": "Gini index"},
+             {"text": "ReliefF"},
+             {"text": "Fast Correlation Based Filter"}
```

Comment:
Best just end this line with a comma as well so it won't need changing when the list is added to. Python doesn't complain about a trailing comma.